Typo in peer dependency version causes npm error since 3.0.10

Hi everyone,

in version 3.0.10, the peer dependency version for eslint in the package.json was changed to
```json
"eslint": "^8.56.0 | ^9.0.0"
```
The 2 alternative versions are separated by a single "|" instead of "||".
As a result, the dependency cannot be resolved (e.g. when running npm update):

```shell
npm error code ERESOLVE
npm error ERESOLVE unable to resolve dependency tree
...
npm error Found: eslint@8.57.0
...
npm error Could not resolve dependency:
npm error peer eslint"@8.56.0 | ^9.0.0" from eslint-plugin-ft-flow@3.0.10
```

Best regards
Martin

--- a/package.json
+++ b/package.json
@@ -59,7 +59,7 @@
     "rimraf": "^3.0.2"
   },
   "peerDependencies": {
-    "eslint": "^8.56.0 | ^9.0.0",
+    "eslint": "^8.56.0 || ^9.0.0",
     "hermes-eslint": ">=0.15.0"
   },
   "keywords": [

